docs/runner/components/subjects_delete.rst

```diff
@@ -0,0 +1,20 @@
+.. _subjects_delete_yamlref:
```

Comment:
This seems entirely unrelated to the purpose of the PR - Is this here by mistake? If not, please can you mention it in the description.

Reply:
I have added to commit message